### ISSUE TITLE
Oracle autocompletes

### ISSUE THREAD

**Issue by LethalPigeon7** [Cockatrice/Cockatrice#2145]
<b>OS:</b>
Win 7 SP1
<b>Cockatrice version:</b>
Version a3a6065 (2016-08-20)
___
I just ran oracle just so I could look at where it downloads from, once I got to that stage(#2) and closed the program it auto-completed. Or at least claimed to with "Update completed successfully, cockatrice will now reload the database."

**Comment by ctrlaltca** [comment]
You ran oracle from inside cockatrice.
Cockatrice saw that oracle was closed, so it reloaded its database supposing that the card download did complete.
The message "Update completed successfully, cockatrice will now reload the database." is misleading, and has been popped up by cockatrice, not by oracle.

**Comment by LethalPigeon7** [comment]
Misleading message, assumed as much. Thank you.

**Comment by tooomm** [comment]
Yeah, i realized that as well... that message is definitely misleading!

A more general message... or a proper way to tell Cockatrice that it (Oracle) aborted or actually finished would be great.

**Comment by ZeldaZach** [comment]
Maybe include a 2nd alert for close()?
